include/rapidjson/reader.h

```diff
@@ -1039,7 +1040,17 @@ class GenericReader {
             }
             else if (RAPIDJSON_UNLIKELY(static_cast<unsigned>(c) < 0x20)) { // RFC 4627: unescaped = %x20-21 / %x23-5B / %x5D-10FFFF
                 if (c == '\0')
-                    RAPIDJSON_PARSE_ERROR(kParseErrorStringMissQuotationMark, is.Tell());
+					RAPIDJSON_PARSE_ERROR(kParseErrorStringMissQuotationMark, is.Tell());
+				else if (!isKey && c == '\n' && (parseFlags & kParseMultiLineValueFlag)) {
```

Comment:
Not sure if we need to support other line endings like `\r` - how does the previous json loading code handle that?